packages/shared-base/src/SingletonModal/index.ts

```diff
@@ -134,7 +134,7 @@ export class SingletonModalQueued<OpenProps = void, CloseProps = void> extends S
         })
     }
 
-    override open(props: OpenProps) {
+    override open = (props: OpenProps) => {
         if (!this.opened) {
             super.open(props)
```

Comment:
Does `super` still work?

Reply:
If the parent class `open` is defined by class field, it does not work.